tasks/ingress-nginx.yml

```diff
@@ -0,0 +1,7 @@
+- name: Copy ingress-nginx files to first server
+  ansible.builtin.template:
+    src: "templates/ingress-nginx-config.yml.j2"
+    dest: "{{ rke2_data_path }}/server/manifests/rke2-ingress-nginx-config.yaml"
+    owner: root
+    group: root
+    mode: 0664
```

Comment:
Please add new line at end of the line. See https://github.com/lablabs/ansible-role-rke2/actions/runs/9078504650/job/25340508973?pr=216

Reply:
Done. Also created https://github.com/lablabs/ansible-role-rke2/pull/222 to fix this issue automatically.